src/fondue/Fondue.js

```diff
@@ -48,6 +53,14 @@ export default class Fondue {
 		);
 	}
 
+	get charCount() {
+		return this.supportedCharacters.length;
+	}
+
+	get glyphCount() {
+		return this._font.opentype.tables.maxp.numGlyphs;
+	}
+
```

Comment:
Can you add tests for these functions?

Reply:
I knew I forgot something that I forgot the previous 17 times also 🙈